src/commands/synthetics/reporters/junit.ts

```diff
@@ -370,11 +370,11 @@ export class JUnitReporter implements Reporter {
       properties: {
         property: [
           {$: {name: 'check_id', value: result.test.public_id}},
-          ...('device' in result.result
+          ...(isDeviceIdSet(result.result)
             ? [
-                {$: {name: 'device', value: result.result.device.id}},
-                {$: {name: 'width', value: result.result.device.width}},
-                {$: {name: 'height', value: result.result.device.height}},
+                {$: {name: 'device', value: result.result.device?.id}},
+                {$: {name: 'width', value: result.result.device?.width}},
+                {$: {name: 'height', value: result.result.device?.height}},
```

Comment:
I believe we don't need the optional chaining  here since `result.result` should be considered a `Required<BrowserServerResult>`?

Reply:
Exactly! That's a git conflict mistake... I purposely used type guards to remove those optional chaining 😅
Thanks for spotting it!